packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -397,6 +397,13 @@ export class GarbageCollector implements IGarbageCollector {
 
         // If session expiry is enabled, we need to close the container when the timeout expires
         if (this.sessionExpiryTimeoutMs !== undefined) {
+            // If Test Override config is set, override Session Expiry timeout
+            const sessionExpiryTimeoutDays =
```

Comment:
Should we make this ms not days? I am thinking from testing perspective where we may not want to wait days to test it and want to have greater control. We could set it to a few seconds or minutes and see it in action along with testing sweep.

Reply:
Ok sure